web/ui/react-app/src/types/PathPrefixProps.ts

```diff
@@ -1,5 +1,6 @@
 interface PathPrefixProps {
   pathPrefix?: string;
+  consolesLink?: string | null;
```

Comment:
As an aside, I don't think `consolesLink` belongs into `PathPrefixProps`, since it doesn't relate to the path prefix, and is only needed by the Nav (and App for pass-through). Since it's only required in two places, I would probably not even make a shared type for it, but just make separate props types for the App and the Nav (in the files where those components live), that each expect this field?